.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "4"
```

Comment:
Yes We can use 7 . 
 4 mean latest 4.x release
7 mean latest 7.x release
Will update PR for 7